src/sensitivities/linalg/symmetric.jl

```diff
@@ -0,0 +1,4 @@
+import Base.Symmetric
+@explicit_intercepts Symmetric Tuple{∇Array}
+∇(::typeof(Symmetric), ::Type{Arg{1}}, p, Y::∇Array, Ȳ::∇Array, X::∇Array) =
+    full(UpperTriangular(Ȳ) + LowerTriangular(Ȳ)') - Diagonal(Ȳ)
```

Comment:
Do we actually need to call `full` here? Would it not be sufficient to return an `UpperTriangular` matrix? Subtracting a `Diagonal` matrix from an `UpperTriangular` matrix still yields an `UpperTriangular` matrix as you would hope, so there's not fiddly that needs to happen.

Reply:
Probably not, no. Removed it in the latest commit.